src/routes/create-klage.tsx

```diff
@@ -30,8 +31,17 @@ const CreateKlage = () => {
 
         const title = getTitle(query, temaKey);
         const saksnummer = getQueryValue(query.saksnummer);
+        const fullmaktsgiver = getQueryValue(query.fullmaktsgiver);
 
-        getDraftKlage(temaKey, title, saksnummer)
+        if (fullmaktsgiver) {
+            getFullmaktsgiver(temaKey, fullmaktsgiver)
+                .then(setFullmaktsgiver)
+                .catch(() => {
+                    setError(finneFullmaktsgiverError(fullmaktsgiver));
```

Comment:
Når denne ender i `catch`, så vil det si at det er noe galt med fullmakten. Mest sannsynlig at den ikke finnes. Stemmer ikke det?

Skal vi i så fall fortsatt opprette/gjenoppta klagen , som logikken under her gjør?

Reply:
> Når denne ender i `catch`, så vil det si at det er noe galt med fullmakten. Mest sannsynlig at den ikke finnes. Stemmer ikke det?
> 
> Skal vi i så fall fortsatt opprette/gjenoppta klagen , som logikken under her gjør?

Jo, det stemmer. Du mener at vi ikke burde hente draft/sette klage hvis fullmaktsgiver ikke kan hentes? Prøver å finne litt ut av logikken her @cskrov

Reply:
Ingen solid mening ennå, men det kan se ut for meg som om dette er mulig:
1) Forsøke å opprette en klage med `fullmaktsgiver` satt.
2) `getFullmaktsgiver` feiler og setter en error.
3) Klagen opprettes med `fullmaktsgiver` satt, selv om `getFullmaktsgiver` feilet.
4) Bruker har nå fått opprettet en klage på vegne av noen de ikke har fullmakt fra.

Dette stoppes forhåpentligvis og mest sannsynlig i backend, men burde vi ikke stoppe dersom `getFullmaktsgiver` feiler?